hindi/swami-vivekananda/teachings-of-swami-vivekananda/quotes.md

```diff
@@ -15,4 +15,9 @@
 4.  उठो, जागो और तब तक रुको नही जब तक मंजिल प्राप्त न हो जाये ।
 
 
-[और देखिये](http://www.hindisahityadarpan.in/2011/11/great-quotations-by-swami-vivekananda.html)
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.